src/nft/components/bag/MobileHoverBag.css.ts

```diff
@@ -8,7 +8,7 @@ export const bagContainer = style([
     bottom: '72',
     left: '16',
     right: '16',
-    background: 'backgroundModule',
+    background: 'surface2',
```

Comment:
Mobile Hover Bag is now very flat with the background, should it have a border like the nav menu?
![Screenshot 2023-08-16 at 11 08 57 AM](https://github.com/Uniswap/interface/assets/11512321/6a943977-08ba-4dd4-8a85-b6f43fef0422)

Reply:
yes! good catch please add border (surface3)

Reply:
cc: @natew

Reply:
![CleanShot 2023-08-22 at 10 58 14](https://github.com/Uniswap/interface/assets/12100/2c165436-6c6c-4363-abbe-c6a08b8ba1a5)

added